src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -86,6 +86,9 @@ public void getSalaryInfoByMonth() {
             String actualResult = salary.getSalaryInfo(sampleNames, scriptArray, dates[0], dates[i]);
             String expectedResult = SalaryInfoTest.exceptedReports[i];
 
+            actualResult = actualResult.replaceAll("\\R", "");
+            expectedResult = expectedResult.replaceAll("\\R", "");
```

Comment:
revert test changes

Reply:
Thank you for this review, but unfortunately, when I use the 'mvn clean package' command, I get an error message that says: ![Screen Shot 2023-07-28 at 17 11 26](https://github.com/mate-academy/jv-salary-info/assets/102083412/876b664f-04f7-44b8-8f18-e937d8443062) . I have tried everything and found that the issue is not in my code because the error indicates that the system doesn't recognize the line separator. I am using "System.lineSeparator" in the code and even have tried "\n," but it always gives the same error. So, I had to add these lines. What should I do? Problem is not in my code.

Reply:
Here is the comparison of the expected (by test) and actual (your) result 
(it is failed because of the extra end line)

![image](https://github.com/mate-academy/jv-salary-info/assets/109764850/d1e867cb-92a7-4f45-a622-adbc650b283b)

Reply:
not fixed